src/libraries/System.Runtime.Loader/ref/System.Runtime.Loader.cs

```diff
@@ -10,6 +10,7 @@ public static partial class AssemblyExtensions
     {
         [System.CLSCompliantAttribute(false)]
         public unsafe static bool TryGetRawMetadata(this System.Reflection.Assembly assembly, out byte* blob, out int length) { throw null; }
+        public static void ApplyUpdate(System.Reflection.Assembly assembly, ReadOnlySpan<byte> metadataDelta, ReadOnlySpan<byte> ilDelta, ReadOnlySpan<byte> pdbDelta = default) {  }
```

Comment:
I assume this is just temporary as https://github.com/dotnet/runtime/pull/48366 is adding this as well (but slightly different).

Reply:
Yep.  Once the other PR goes in, this will be removed.